src/hooks/UseForm/useForm/utils/internal.ts

```diff
@@ -0,0 +1,175 @@
+import React from 'react'
+import { CoordinatesOrNever, Tuple } from '../../../UseGraph/Graph'
+import {
+  FieldData,
+  FieldElement,
+  Fields,
+  FieldsData,
+  FormData,
+  PartialDataKeys,
+  RegisterResult,
+  isCheckboxInput,
+  isDateInput,
+  isFileInput,
+  isNumberInput,
+  isRadioInput,
+} from '../types'
+
+export const getElementDefaultValue = <TData extends FieldsData>(
+  // the type does not matter since we check for each property
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  element: any,
```

Comment:
There was an issue where selects didn't have a `defaultValue` but I went ahead and added logic for single/multiple selects so that is no longer an issue and the types are updated :)